### PR TITLE
Adds new Trait book and Loadout to the Preacher.

### PR DESCRIPTION
**Pull request by Greg-Joffer**
## About The Pull Request

Right, since we cannot add priestesses into the legion, I took the prerogative to added it to the preacher role, giving it a bit of flavour but really no combat options. Added lessons of mars back as a trait book for the loadout- this is a trait all the legion spawn with.
## Pre-Merge Checklist
- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [X] You documented all of your changes.

## Changelog
:cl:
WH0_T00Kthejam
Added a new trait book : Lessons of Mars
Added a new Preacher loadout : Priest of Mars.
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/wasteland.dm [hunk 1]
```diff
@@ -1362,8 +1362,13 @@ Raider
...
 		///obj/item/ammo_box/m44 = 2
-	)
-
+	
+	/datum/outfit/loadout/priest_of_mars
+	name = "Priest of mars"
+	backpack_contents = list(
+	/obj/item/clothing/under/f13/pmars_robe	= 1,	//this is for the asthetic add it to orator if you want
+	/obj/item/book/granter/legionalchemy = 1	//gives them the same trait auxillas have
+)
```
Comment (Zero-Percent-Angel) [inline code/modules/jobs/job_types/wasteland.dm:1371]: I'd bring this in a tab

File: code/modules/jobs/job_types/wasteland.dm [hunk 2]
```diff
@@ -1362,8 +1362,13 @@ Raider
 		/obj/item/grenade/homemade/coffeepotbomb = 2,	//This is funny. I swear guys.
 		///obj/item/gun/ballistic/revolver/m29 = 1,		//Moved here for more *variety*
 		///obj/item/ammo_box/m44 = 2
-	)
-
+	
+	/datum/outfit/loadout/priest_of_mars
```
Comment (Zero-Percent-Angel) [inline code/modules/jobs/job_types/wasteland.dm:1366]: Take this tab at the start of the line out!

File: code/modules/jobs/job_types/wasteland.dm [hunk 3]
```diff
@@ -1362,8 +1362,12 @@ Raider
 		/obj/item/grenade/homemade/coffeepotbomb = 2,	//This is funny. I swear guys.
 		///obj/item/gun/ballistic/revolver/m29 = 1,		//Moved here for more *variety*
 		///obj/item/ammo_box/m44 = 2
-	)
```
Comment (Zero-Percent-Angel) [inline code/modules/jobs/job_types/wasteland.dm:1365]: You need to add this back in

Comment (Zero-Percent-Angel) [reply code/modules/jobs/job_types/wasteland.dm:1365]: Preferably at the correct indentation!

**Comment by Zero-Percent-Angel** [comment]
Discussed with contributor closing this!

**Comment by DorfGuy1** [comment]
why no priestesses

**Pull request closed without merging**